### DISCUSSION TITLE
is there way to call intention action such as 'add format string argument' in ideavim?

### DISCUSSION THREAD

**Discussion by woodgear** [JetBrains/ideavim#414] [Q&A]
https://www.jetbrains.com/help/go/intention-actions.html#examples-of-intention-actions-for-declarations

**Comment by AlexPl292** [comment]
Hi, we don't have direct integrations with intentions at the moment, but you can check if some intentions have a separation action id by enabling "track action IDs" and executing this action: https://github.com/JetBrains/ideavim#finding-action-ids
